MovieAppPetProject/MovieAppPetProject/Login/LoginViewController.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LoginViewController.swift
+//  MovieAppPetProject
+//
+//  Created by Sebastian Jacobs on 2024/04/15.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController, LoginViewModelDelegate {
+
+    // MARK: - Outlets
+    @IBOutlet private weak var usernameInput: UITextField!
+    @IBOutlet private weak var passwordInput: UITextField!
+
+    // MARK: - Variable
+    private var loginViewModel: LoginViewModel!
+
+    // MARK: - Functions
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        loginViewModel = LoginViewModel(delegate: self)
+        usernameInput.attributedPlaceholder = NSAttributedString(string: "Username", attributes: [NSAttributedString.Key.foregroundColor: UIColor.gray])
```

Comment:
Please put these in a private function and call it in the viewDidLoad for neatness.
Maybe called setUpPlaceHolderText()

Reply:
changes made in login cleanup PR